js/components/MilkdropWindow/Presets.js

```diff
@@ -72,6 +96,20 @@ export default class Presets {
     return this._selectIndex(idx);
   }
 
+  async _convertPreset(file) {
+    return new Promise((resolve, reject) => {
+      require.ensure(
+        ["milkdrop-preset-converter-aws"],
+        async require => {
+          const { convertPreset } = require("milkdrop-preset-converter-aws");
+          resolve(convertPreset(file));
```

Comment:
What happens if this fails? Could happen if the network is down, or probably for any number of reasons (AWS is down, bad .milk file etc).

Reply:
I think our convention (for now) is to pop an alert. Not beautiful, but it's a start. We should do that here as well.

Reply:
OK, sounds good